packages/openapi3/src/cli/actions/convert/interfaces.ts

```diff
@@ -0,0 +1,103 @@
+import { Contact, License } from "@typespec/openapi";
+import {
+  OpenAPI3Encoding,
+  OpenAPI3Info,
+  OpenAPI3Responses,
+  OpenAPI3Schema,
+  Refable,
+} from "../../../types.js";
+
+export type FullOpenAPI3Info = OpenAPI3Info & {
```

Comment:
Should maybe update the `OpenAPI3Info` type no?

The types in types.ts are not necessarily completely accurate